aws_lambda_powertools/utilities/parser/parser.py

```diff
@@ -84,7 +86,7 @@ def handler(event: Order, context: LambdaContext):
     return handler(parsed_event, context)
 
 
-def parse(event: Dict[str, Any], model: BaseModel, envelope: Optional[BaseEnvelope] = None) -> Any:
+def parse(event: Dict[str, Any], model: Type[Model], envelope: Optional[Type[Envelope]] = None) -> Any:
```

Comment:
Will this method return `Any` object?
If the method only returns the parsed model then the type should be `Model`.